modules/mod_ginger_rdf/mod_ginger_rdf.erl

```diff
@@ -58,7 +48,17 @@ manage_schema(_, Context) ->
                 Context
```

Comment:
Yes indeed, the predicate update above is a tricky sideeffect.

Reply:
These changes are yours! (I built this PR on top of that branch, and it took these two commits with it)

Reply:
Oh haha! :D Now you say it..I recall doing something similar indeed :joy: